docs/configuration/data-validation.md

```diff
@@ -108,3 +109,34 @@ PROTECTION_RULES = {
     ]
 }
 ```
+
+### Configuration via Web UI
+
+![image](https://github.com/netbox-community/netbox/assets/46369917/6fb69ab8-1f13-4b86-b84c-ce697af8f679)
```

Comment:
All images presented in the documentation must be included in the code base.

Reply:
@LHBL2003 you actually need to include the image in the codebase and git-add it to the code so that it appears in the PR.

Reply:
OK feels strange. But this means that the image is displayed as a path. :)

Reply:
@arthanson I'm not sure if I understand you correctly. Do I have to insert the image paths "![ Image ] ( https://..." into the code block ```python... ![ Image ] ( https://...```. Or do you think I have to upload the images as an attachment on the Git Hub page so that they get an image ID "![IMG_0047](https://".
Sorry for the stupid question, I have now done the latter.